lib/config/services.js

```diff
@@ -23,22 +25,22 @@ const {
 exports.services = {
     enableHousekeeping: process.env?.ENABLE_HOUSEKEEPING ?? false,
     aliEcsGui: {
-        url: process.env?.ALI_ECS_GUI_URL || changeOriginPort(origin, 8080),
-        token: process.env?.ALI_ECS_GUI_TOKEN,
+        url: process.env?.ALI_ECS_GUI_URL || null,
```

Comment:
You should use ?? instead of ||

Reply:
I'd say no, because if, in bash, you have statement like `export VAR=` then `process.env.VAR` evaluate it to empty string which remains if using `??`